src/components/error/index.js

```diff
@@ -0,0 +1,8 @@
+import classes from './styles.module.css';
+
+const Error = props => {
+    const { message } = props;
+    return <p className={classes.errorContent}>{message}</p>;
```

Comment:
Note: will be showing the error code as well, like a faded text in the background

Reply:
Done